tests/class-unittestsbootstrap.php

```diff
@@ -69,8 +70,10 @@ public function set_path_props() {
 	 * Set server props
 	 */
 	public function set_server_props() {
+		// phpcs:disable WordPress.Security.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
Do you think we can sanitize the input here instead of ignoring the line?

Reply:
Since this file is used for PHPUnit testing, I thought it would be okay to skip the checks.

I revisited its logic once and found that after the rewrite it didn't have to skip checking. Adjusted in https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/446/commits/db107f7d216adbd9798695e9c8bd05ddce288a67.